src/main/java/org/opentripplanner/routing/api/request/preference/RoutingPreferences.java

```diff
@@ -95,6 +95,13 @@ public VehicleRentalPreferences rental() {
     return rental;
   }
 
+  /**
+   * Get parking preferences for the traverse mode. Note, only car and bike are supported.
+   */
+  public VehicleParkingPreferences parking(TraverseMode mode) {
```

Comment:
Note! The `TraverseMode` is not part of the api, and using it here create a circular dependency - this is nothing new, so I am not requesting you to change it - but this is adding a string to the spagetti.